ci/run.sh

```diff
@@ -3,7 +3,13 @@
 set -xeu
 set -o pipefail
 
-dchatty=1 dub test -b unittest-cov --skip-registry=all --compiler=${DC}
+dchatty=1 timeout -v -s SEGV 12m dub test -b unittest-cov --skip-registry=all --compiler=${DC}
```

Comment:
It might actually be useful to build the unittest binary and run this binary rather than dub. That way when we  segfault it we could generate a core dump. (also needs `ulimit -u unlimited` set).

So:

```
dub build -c unittest -b unittest-cov --skip-registry=all --compiler=${DC}
dchatty=1 timeout -v -s SEGV 12m ./build/agora-unittests
```

What do you think?

Reply:
Or would it make no difference?